src/settings-ui/Settings.UI/ViewModels/AdvancedPasteViewModel.cs

```diff
@@ -131,6 +142,16 @@ public bool IsEnabledGpoConfigured
             get => _enabledStateIsGPOConfigured;
         }
 
+        public bool IsOnlineAIModelsGpoConfigured
+        {
+            get => _onlineAIModelsIsGPOConfigured || _enabledGpoRuleConfiguration == GpoRuleConfigured.Disabled;
+        }
+
+        public bool ShowOnlineAIModelsGpoConfiguredInfoBar
+        {
+            get => _onlineAIModelsIsGPOConfigured && _enabledGpoRuleConfiguration != GpoRuleConfigured.Disabled;
```

Comment:
```suggestion
            get => _onlineAIModelsIsGPOConfiguredToDisableAI && _enabledGpoRuleConfiguration != GpoRuleConfigured.Disabled;
```

For better clarification we should make the var name more precise.